code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -188,7 +188,7 @@
 				for (var/mob/M in viewers(src, null))
 					M.show_message("<span class='danger'>[user] hits [src] with [O], however [src] is too armored.</span>")
 			else
-				to_chat(usr, "<span class='warning'>[src] is wearing too much armor. You can't reach \his skin.</span>")
+				to_chat(usr, "<span class='warning'>[src] is wearing too much armor. You can't reach his skin.</span>")
```

Comment:
This one was fine as-is since `[src]` was the last embedded object

Reply:
woopsie, swapped it back